packages/extension-browser/src/devtools/panel/views/partials/hint.ts

```diff
@@ -0,0 +1,32 @@
+import html from '../../../../shared/html-literal';
+import { HintResults } from '../../../../shared/types';
+
+import problemView from './problem';
+
+import './hint.css';
+
+export default function view({ name, problems, helpURL }: HintResults) {
+
+    const hintStatus = !problems.length ?
+        'PASSED' :
+        `${problems.length} hint${problems.length !== 1 ? 's' : ''}`;
+
+    return html`
+        <details class="hint">
+            <summary class="hint__summary">
+                <span class="hint__name">
+                    ${name}:
```

Comment:
You changed this recently, right? Last time I tried it was using the id I think.

Reply:
Currently this is still the ID, but is set by the formatter so when the new hint metadata is available there's only one spot that needs to change to cut this over to the name:

In `content-script/formatter.ts`:
```ts
    private buildHintResults(resources: HintResources, problems: Problem[], category: string): HintResults[] {
        return this.getHints(resources, category).map((id) => {
            return {
                helpURL: `https://webhint.io/docs/user-guide/hints/hint-${id}/`,
                id,
                name: id, // TODO: Map to a user-friendly name.
                problems: problems.filter((problem) => {
                    return id === problem.hintId;
                })
            };
        });
    }
```